### ISSUE TITLE
Maps for Dungeons

### ISSUE THREAD

**Issue by kefirok** [obble/modui#93]
Works bad. 
- Need add levels on the map(some dungeons have several maps for one on retail).
- Make right click to World map

**Comment by obble** [comment]
yeah, I've been meaning to add an option to manually select through dungeon levels via a dropdown — they are all there and are currently triggered by moving to a specific named sub-zone of the dungeon (the only real way to register whereabouts in the dungeon you are) but some are a little temperamental, the deadmines in particular.

right-click to access world map is a good idea too.
